7-TimeSeries/2-ARIMA/README.md

```diff
@@ -223,10 +223,10 @@ Walk-forward validation is the gold standard of time series model evaluation and
 
     ```python
     test_shifted = test.copy()
-    
-    for t in range(1, HORIZON):
+
+    for t in range(1, HORIZON+1):
```

Comment:
This is the relevant change

Reply:
let's add it as well to the /solutions/notebook.ipynb file! thank you!

Reply:
I'll do it, thank you Alfredo! :)